cassovary-core/src/main/scala/com/twitter/cassovary/algorithms/centrality/ClosenessCentrality.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2015 Twitter, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.twitter.cassovary.algorithms.centrality
+
+import com.twitter.cassovary.graph.{GraphDir, BreadthFirstTraverser, Walk, DirectedGraph}
+import scala.collection.immutable
+
+/**
+ * Calculate the closeness centrality of {@code graph}.
```

Comment:
Even though the current code is littered with {@code ...} we don't use it anymore. Prefer `graph`